docs/aws/audit/elbmonitoring/rules/classic_elb_in_use.mdx

```diff
@@ -24,6 +24,219 @@ CBP
+### Check Cause
+<AccordionGroup>
+<Accordion title='Using Console' defaultOpen='true'>
+1. Sign in to the AWS Management Console and open the Amazon EC2 console at https://console.aws.amazon.com/ec2/.
+
+2. In the navigation pane, under LOAD BALANCING, choose Load Balancers.
+
+3. In the Load Balancer type column, you can see the type of each load balancer. If any of them are listed as "Classic Load Balancer", it indicates that a Classic ELB is in use.
+
+4. For a more detailed view, select the Classic Load Balancer. In the Description tab below, you can see more details about the selected load balancer.
+</Accordion>
+
+<Accordion title='Using CLI'>
+1. Install and configure AWS CLI: Before you can start using AWS CLI, you need to install it on your local system. You can download it from the official AWS website. After installation, you need to configure it with your AWS account credentials. You can do this by running the command `aws configure` and then entering your AWS Access Key ID, Secret Access Key, Default region name, and Default output format when prompted.
+
+2. List all Classic Load Balancers: Run the following command to list all the Classic Load Balancers in your AWS account.
+
+   ```
+   aws elb describe-load-balancers
+   ```
+
+   This command will return a JSON output with the details of all the Classic Load Balancers.
+
+3. Parse the output: You can parse the output using a tool like jq to get the names of the Classic Load Balancers. Here is an example command:
+
+   ```
+   aws elb describe-load-balancers | jq -r '.LoadBalancerDescriptions[].LoadBalancerName'
+   ```
+
+4. Check for Classic Load Balancers: If the above command returns any names, it means that there are Classic Load Balancers in use. If it doesn't return any names, it means that there are no Classic Load Balancers in use.
+</Accordion>
+
+<Accordion title='Using Python'>
+1. Install the necessary AWS SDK for Python (Boto3) if you haven't done so already. You can install it using pip:
+
+```python
+pip install boto3
+```
+
+2. Import the necessary modules and create a session using your AWS credentials:
+
+```python
+import boto3
+
+session = boto3.Session(
+    aws_access_key_id='YOUR_ACCESS_KEY',
+    aws_secret_access_key='YOUR_SECRET_KEY',
+    region_name='YOUR_REGION'
+)
+```
+
+3. Create an ELB client using the session:
+
+```python
+elb = session.client('elbv2')
+```
+
+4. Now, you can use the `describe_load_balancers` function to get a list of all load balancers. Then, iterate over the list and check the `Type` of each load balancer. If the type is `classic`, then it's a misconfiguration:
+
+```python
+response = elb.describe_load_balancers()
+
+for lb in response['LoadBalancers']:
+    if lb['Type'] == 'classic':
+        print(f"Misconfiguration detected: Classic ELB in use - {lb['LoadBalancerName']}")
+```
+
+This script will print the names of all classic ELBs in use. If there are no classic ELBs, it won't print anything.
+</Accordion>
+
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target='_blank'>Click here for more details</a></b>